packages/web-wallet/src/types/credential/index.ts

```diff
@@ -72,20 +75,21 @@ export class CredentialTableItem {
     this.miniCardView = data.credentialCardViewProps
   }
 
-  static from(credential: Credential, parties: Party[], credentialSummary?: CredentialSummary): CredentialTableItem {
-    const issuerPartyIdentity = getMatchingIdentity(parties, credential.issuerDid)
+  static from(credential: DigitalCredential, parties: Party[], credentialSummary?: CredentialSummary): CredentialTableItem {
+    const issuerPartyIdentity = getMatchingIdentity(parties, credential.issuerCorrelationId)
     if (!issuerPartyIdentity) {
-      throw new Error(`Couldn't find matching identity for the issuer: ${credential.issuerDid}`)
+      throw new Error(`Couldn't find matching identity for the issuer: ${credential.issuerCorrelationId}`)
     }
 
-    const subjectPartyIdentity = credential.subjectDid ? getMatchingIdentity(parties, credential.subjectDid) : undefined
+    const subjectPartyIdentity = credential.subjectCorrelationId ? getMatchingIdentity(parties, credential.subjectCorrelationId) : undefined
     const subjectParty = subjectPartyIdentity ? subjectPartyIdentity.party : undefined
 
-    const issuanceDateStr = formatDate(credential.issuanceDate)
-    const expirationDateStr = formatDate(credential.expirationDate)
+    const issuanceDateStr = formatDate(credential.createdAt as unknown as string) // FIXME use other REST client
```

Comment:
This is not the issuance data of course. That is validFrom. Don't mix db entity dates with actual dates about issuance and expiration

Reply:
Used validFrom for now